src/django_healthchecks/checker.py

```diff
@@ -73,6 +78,24 @@ def _get_registered_health_checks():
     return getattr(settings, 'HEALTH_CHECKS', {})
 
 
+def _http_healthcheck_func(url):
+    def handle_remote_request():
+        try:
+            response = requests.get(url, timeout=_get_http_healthcheck_timeout())
+        except requests.exceptions.RequestException:
+            return False
```

Comment:
check for socket errors / SSL errors? (`except IOError` catches everything on Python 3, but less on Python 2)